pkg/kubelet/metrics/metrics.go

```diff
@@ -85,6 +86,7 @@ const (
 )
 
 var (
+	// NodeName is a Gauge that tracks the name of the assigned via the label "node_key".
```

Comment:
I'm a little worried about any discrepancies between these variable comments and the `Help` text. For example, for `NodeName`, should our comment instead be `// NodeName is a Gauge that tracks the node's name. The count is always 1.`?

Reply:
I was wondering about the same. I'll just use the help text for the comment then.